assets/i18n/hi.json

```diff
@@ -0,0 +1,276 @@
+  "I18N_PLAYER_STAY_ANONYMOUS": "गुमनाम रहे",
+  "I18N_PLAYER_SUBMIT_BUTTON": "जमा करें",
+  "I18N_PLAYER_TAGS_TOOLTIP": "टैग",
+  "I18N_PLAYER_THANK_FEEDBACK": "प्रतिक्रिया के लिए आपका धन्यवाद!",
+  "I18N_PLAYER_UNRATED": "अनरेटेड",
+  "I18N_PLAYER_VIEWS_TOOLTIP": "मत",
+  "I18N_PREFERENCES_BIO": "जैव",
+  "I18N_PREFERENCES_BIO_EXPLAIN_TEXT": "यह फ़ील्ड वैकल्पिक है। यहाँ कुछ भी लिखा हुआ सार्वजनिक है और पूरा विश्व देख सकता है ।",
+  "I18N_PREFERENCES_BREADCRUMB": "तरजीह",
+  "I18N_PREFERENCES_CANCEL_BUTTON": "रद्द करे",
+  "I18N_PREFERENCES_CHANGE_PICTURE": "प्रोफ़ाइल चित्र बदलें",
+  "I18N_PREFERENCES_EMAIL": "ईमेल",
+  "I18N_PREFERENCES_EMAIL_EXPLAIN": "केवल मध्यस्थों और साइट व्यवस्थापक यह देख सकते हैं ।",
+  "I18N_PREFERENCES_EMAIL_RECEIVE_NEWS": "साइट के बारे में समाचार प्राप्त करे।",
+  "I18N_PREFERENCES_HEADING": "प्राथमिकताएँ",
+  "I18N_PREFERENCES_HEADING_SUBTEXT": "आपके द्वारा की गयी किसी भी प्रकार की परिवर्तन इस पेज पर ऑटो सेव हो जाएगी। ",
+  "I18N_PREFERENCES_PAGE_TITLE": "अपने प्रोफ़ाइल प्राथमिकताएँ बदले - Oppia",
+  "I18N_PREFERENCES_PICTURE": "चित्र",
+  "I18N_PREFERENCES_PREFERRED_EXPLORATION_LANGUAGE": "पसंदीदा एक्सप्लोरेशन की भाषा",
+  "I18N_PREFERENCES_PREFERRED_EXPLORATION_LANGUAGE_EXPLAIN": "यह भाषाएँ डिफ़ॉल्ट रूप से चयन कि जाती है जब आप एक्सप्लोरेशन के लिए गैलरी कि खोज करते हैं।",
+  "I18N_PREFERENCES_PREFERRED_SITE_LANGUAGE": "पसंदीदा साइट की भाषा",
+  "I18N_PREFERENCES_PREFERRED_SITE_LANGUAGE_EXPLAIN": "इस भाषा में साइट दिखाया जायेगा।",
+  "I18N_PREFERENCES_PREFERRED_SITE_LANGUAGE_PLACEHOLDER": "पसंदीदा साइट की भाषा",
+  "I18N_PREFERENCES_PROFILE_PICTURE_ADD": "प्रोफाइल पिक्चर अपलोड करे",
+  "I18N_PREFERENCES_PROFILE_PICTURE_DRAG": "खींच कर आकार अथवा साइज बदले:",
+  "I18N_PREFERENCES_PROFILE_PICTURE_ERROR": "त्रुटि: यह पिक्चर अपलोड नहीं की जा सकती है।",
+  "I18N_PREFERENCES_PROFILE_PICTURE_UPLOAD": "प्रोफाइल पिक्चर अपलोड करे",
+  "I18N_PREFERENCES_SELECT_EXPLORATION_LANGUAGE": "अपनी पसंदीदा भाषा चुने...",
+  "I18N_PREFERENCES_SUBJECT_INTERESTS": "विषय हेतु रूचि",
+  "I18N_PREFERENCES_SUBJECT_INTERESTS_HELP_BLOCK": "उदाहरण: अंक शास्त्र, कंप्यूटर विज्ञान, कला, ...",
+  "I18N_PREFERENCES_SUBJECT_INTERESTS_INVALID_SEARCH": "नयी विषय हेतु रूचि प्रदान करे (छोटे अक्षरों और रिक्त स्थान का उपयोग करे)...",
+  "I18N_PREFERENCES_SUBJECT_INTERESTS_PLACEHOLDER": "आपकी किस विषय में रूचि है...",
+  "I18N_PREFERENCES_USERNAME": "उपयोगकर्ता नाम",
+  "I18N_PREFERENCES_USERNAME_NOT_SELECTED": "अभी तक चयनित नहीं किया गया है",
+  "I18N_SIDEBAR_ABOUT_LINK": "के बारे में",
+  "I18N_SIDEBAR_BLOG": "ब्लॉग",
+  "I18N_SIDEBAR_CONTACT_US": "संपर्क करें",
+  "I18N_SIDEBAR_FOLLOW_US": "हमसे जुड़े",
+  "I18N_SIDEBAR_FORUM": "मंच",
+  "I18N_SIDEBAR_LIBRARY_LINK": "पुस्तकालय",
+  "I18N_SIDEBAR_TEACH_WITH_OPPIA": "Oppia के साथ सिखाये",
+  "I18N_SIGNUP_AGREE_LICENSE_DESCRIPTION": "इस पाठ के बाईं ओर की बॉक्स को चुन कर, आप <[sitename]> की शर्तों से सहमत हैं और उन्हें स्वीकार करते, जो <a href=\"/terms\" target=\"_blank\">यहाँ उपलब्ध  है </a>.",
```

Comment:
I think the former one is better. The case you are suggesting is something like, you agree to **Oppia's** terms and they are acceptable to you. The former one sounds like, you agree to **Oppia's** terms and accept them.